src/main/java/jp/co/future/uroborosql/store/NioSqlManagerImpl.java

```diff
@@ -0,0 +1,676 @@
+							+ path.toAbsolutePath().toString() + "]");
+				}
+
+				synchronized (sqlName) {
+					try {
+						String body = new String(Files.readAllBytes(path), charset).trim();
+						if (body.endsWith("/") && !body.endsWith("*/")) {
+							body = StringUtils.stripEnd(body, "/");
+						} else {
+							body = body + System.lineSeparator();
+						}
+						sqlBody = body;
+						log.debug("Loaded SQL template.[{}]", path);
+					} catch (IOException e) {
+						throw new UroborosqlRuntimeException("Failed to load SQL template["
+								+ path.toAbsolutePath().toString() + "].", e);
+					}
+				}
+			}
+
+			return sqlBody;
+		}
+
+		/**
+		 * 同じSqlNameになるPathの優先度判定を行い、優先度が高いPathが指定された場合保持しているPathの置き換えを行う
+		 *
+		 * @param newPath 判定用Path
+		 * @param remove 指定した判定用Pathを削除する場合に<code>true</code>を指定
+		 *
+		 * @return 判定後のSqlInfo
+		 */
+		private SqlInfo computePath(final Path newPath, final boolean remove) {
+			synchronized (sqlName) {
+				// 変更前の有効Pathを保持しておく
+				Path oldPath = getPath();
+
+				// 引数で渡された判定用PathをpathListへ追加、またはpathListから削除する
+				if (!pathList.contains(newPath)) {
+					if (!remove) {
+						pathList.add(newPath);
+					}
+				} else {
+					if (remove) {
+						pathList.remove(newPath);
+						if (pathList.isEmpty()) {
+							// pathListが空になった場合はこのSqlInfoをsqlInfosから除外するためにnullを返す
+							return null;
+						}
+					}
+				}
+
+				if (pathList.size() > 1) {
+					// 優先度が高いPathが先頭に来るようにソートを行う
+					// 1. Dialect付Pathを優先
+					// 2. file schemeをjar schemeよりも優先
+					// 3. Path同士のcompare
+					pathList.sort((p1, p2) -> {
+						if (p1 == null && p2 == null) {
+							return 0;
+						} else if (p1 != null && p2 == null) {
+							return -1;
+						} else if (p1 == null && p2 != null) {
+							return 1;
+						}
+
+						// DialectPathの比較
+						boolean p1HasDialect = hasDialect(p1);
+						boolean p2HasDialect = hasDialect(p2);
+
+						if (p1HasDialect && !p2HasDialect) {
+							return -1;
+						} else if (!p1HasDialect && p2HasDialect) {
+							return 1;
+						}
+
+						// schemeの比較
+						String p1Scheme = p1.toUri().getScheme();
+						String p2Scheme = p2.toUri().getScheme();
+
+						if (!p1Scheme.equals(p2Scheme)) {
+							if (p1Scheme.equals(SCHEME_FILE)) {
+								return -1;
+							} else {
+								return 1;
+							}
+						}
+
+						return p1.compareTo(p2);
+					});
+				}
+
+				boolean replaceFlag = false;
+				// ソートによる再計算後の有効Pathを取得する
+				Path currentPath = getPath();
+				FileTime currentTimeStamp = getLastModifiedTime(currentPath);
+				if (!oldPath.equals(currentPath)) {
+					replaceFlag = true;
+					log.debug("sql file switched. sqlName={}, oldPath={}, newPath={}, lastModified={}", sqlName,
+							oldPath, currentPath, currentTimeStamp.toString());
+				} else {
+					if (!this.lastModified.equals(currentTimeStamp)) {
+						replaceFlag = true;
+						log.debug("sql file changed. sqlName={}, path={}, lastModified={}", sqlName, currentPath,
```

Comment:
Although there is an advantage that the file change is notified in a timely manner during REPL execution, I thought that it is better to set it to `log.trace` because the command cursor may carry out unintended execution with it.

As an alternative, I think that it is useful to know the last update date of the SQL file with the REPL `view` command.

Similarly, L658 is also `log.trace`.

Reply:
Change to log.trace.
Since there is no API for acquiring the last update date and time with REPL view command, we will fix it here with another PR.